interfaces/Portal/src/app/services/programs-service-api.service.ts

```diff
@@ -600,6 +604,13 @@ export class ProgramsServiceApiService {
     sort?: PaginationSort,
     // TODO: Fix the 'any' for the 'links' parameter
   ): Promise<{ data: Person[]; meta: PaginationMetadata; links: any }> {
+    const quickSearch = filters
+      ? filters.find(
+          (filter) =>
+            filter.name === this.filterService.DEFAULT_FILTER_OPTION.name,
```

Comment:
On second thought @arsforza ... We should be sharing this name-string via an Enum here and not through the 'whole FilterService`, I think.

To prevent a really complicated dependency-diagram etc.. But enum-values are 'just strings', so there is less complexity necessary.